README.md

```diff
@@ -1,16 +1,129 @@
-# Title
+# Class Extensions and Inheritance
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Use the `extends` keyword
+- Use the `super` method
+- Explain when `class` inheritance should be used in JavaScript
 
 ## Introduction
 
-## SWBAT 1
+In JavaScript, as in other Object Oriented languages, we've learned
+that you can create `class`es and build methods that can perform
+actions on instance data, or specific to the `class`. What if you have
+`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
+and `Bird`, which all have a method for `speak`? In JavaScript, you can
+create "child" object `class`es (constructors) that inherit features
```

Comment:
maybe leave out (constructors). 'you can create "child" object `class`es that inherit features from their "parent" `class`es' seems clear enough